Makefile

```diff
@@ -31,13 +34,16 @@ ifneq ("$(wildcard $(SCSS_DIR320)/gtk-dark.scss)","")
 else
 	cp "$(DIST_DIR320)/gtk.css" "$(DIST_DIR320)/gtk-dark.css"
 endif
-css: css_gtk3 css_gtk320
+css_cinnamon:
+	mkdir -p $(DIST_DIR_CINNAMON)
+	$(SASS) $(SASSFLAGS) "$(SCSS_DIR_CINNAMON)" "$(SCSS_DIR_CINNAMON)/cinnamon.scss" "$(DIST_DIR_CINNAMON)/cinnamon.css"
+css: css_gtk3 css_gtk320 css_cinnamon
 
 gresource_gtk3: css_gtk3
 	$(GLIB_COMPILE_RESOURCES) --sourcedir="$(RES_DIR)" "$(RES_DIR)/gtk.gresource.xml"
 gresource_gtk320: css_gtk320
 	$(GLIB_COMPILE_RESOURCES) --sourcedir="$(RES_DIR320)" "$(RES_DIR320)/gtk.gresource.xml"
-gresource: gresource_gtk3 gresource_gtk320
+gresource: gresource_gtk3 gresource_gtk320 css_cinnamon
```

Comment:
instead add `css_connamon` to `all:` target

Reply:
Done